docs/tracing.md

```diff
@@ -0,0 +1,23 @@
+# Tracing
```

Comment:
It would be nice to include a link to this doc in https://github.com/containerbuildsystem/cachito#more-documentation.

Reply:
Added.